### PR TITLE
Enable vagrant-cachier if it's installed

### PR DESCRIPTION
**Pull request by thedrow**
It's faster to provision the virtual machines if the cachier plugin is installed and enabled.
This PR enables cachier and NFS for cachier.

### CODE REVIEW

**Comment by osqueryer** [comment]
Can one of the osquery admins reply with "ok to test" to kick off a build...

**Comment by facebook-github-bot-6** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-6** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: Vagrantfile [hunk 1]
```diff
@@ -152,15 +160,16 @@ Vagrant.configure("2") do |config|
             ".git/objects",
             ".git/modules/third-party/objects"
           ]
+      else
+        # Private network for NFS
+        build.vm.network :private_network, ip: "192.168.56.101"
```
Comment (theopolis) [inline Vagrantfile]: Do you want to include
```
build.vm.synced_folder ".", "/vagrant", type: "nfs"
```
here too?

Comment (thedrow) [inline Vagrantfile]: Yes.

**Comment by thedrow** [comment]
I still have to test this on my Mac OSX. There's a problem with the apt cache.
Please hold with merging this.

File: Vagrantfile (hunk 1 above)
Comment (theopolis) [inline Vagrantfile]: Ping, if we can get the other NFS related settings moved here too, we can merge this.

**Comment by thedrow** [comment]
Let's postpone this for now. I haven't had the time to fix this.

**Pull request closed without merging**